docker/docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     environment:
       DB_HOST: database
     healthcheck:
-      test: ps aux | grep apache2 | grep -v grep > /dev/null
+      test: curl -s -o /dev/null -w "%{http_code}" http://cdash:8080/ping | grep 200 > /dev/null
```

Comment:
As far as I am aware, `APP_URL` is not used to interpret any incoming connections.  For example, I can ping any route at `localhost://...` or `cdash://...` in the CDash container and it will respond with the same result.  `APP_URL` is important when redirecting to other routes.